specs/commandline/inspect.md

```diff
@@ -0,0 +1,269 @@
+        ├── certificates
+            ├── SHA1 fingerprint: 2f1rr5b8455381frdajc83b4bd305b743cc9513u
+                ├── issued to: Microsoft Root Certificate Authority 2010
+                ├── issued by: Microsoft Root Certificate Authority 2010
+                ├── expiry: Fri Jun 23 22:04:01 2023
+            ├── SHA1 fingerprint: 8BFE3107712B3C886B1C96AAEC89984914DC9B6B
+                ├── issued to: Microsoft Code Signing PCA 2010
+                ├── issued by: Microsoft Root Certificate Authority 2010
+                ├── expiry: Sun Jul 06 20:50:17 2025
+        └── signed attributes 
+            ├── media type: application/vnd.oci.image.manifest.v1+json
+            ├── digest: sha256:b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9
+            └── size: 16724
+```
+
+## Usage signatures on an OCI artifact identified by a tag
+
+```text
+`Tags` are mutable, but `Digests` uniquely and immutably identify an artifact. If a tag is used to identify a signed artifact, notation resolves the tag to the `digest` first.
+```
+
+```shell
+# Prerequisites: Signatures are stored in a registry referencing the signed OCI artifact
+notation inspect localhost:5000/net-monitor:v1
+```
+
+An example output:
+```text
+Resolved artifact tag `v1` to digest `sha256:b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9` before inspect.
+Warning: The resolved digest may not point to the same signed artifact, since tags are mutable.
+```
+
+```shell
+localhost:5000/net-monitor@sha256:ca5427b5567d3e06a72e52d7da7dabfac484efe37a5380ee9088f7ace2eaab9
+└── application/vnd.cncf.notary.signature
+    ├── sha256:aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
+        ├── signing algorithm: RSASSA-PSS-SHA-256
+        ├── signed attributes
+            ├── content type: application/vnd.cncf.notary.payload.v1+json
+            ├── signing scheme: notary.default.x509
+            ├── signing time: Fri Jun 23 22:04:01 2023
+            ├── expiry: Sat Jun 29 22:04:01 2024
+            ├── io.cncf.notary.verificationPlugin: com.example.nv2plugin
+        ├── user defined attributes
+            ├── io.wabbit-networks.buildId: 123
+        ├── unsigned attributes
+            ├── io.cncf.notary.timestampSignature: <Base64(TimeStampToken)>
+            ├── io.cncf.notary.signingAgent: notation/1.0.0
+        ├── certificates
+            ├── SHA1 fingerprint: 2f1cc5b8455381cdefac83b4bd305b789cc9c16e
+                ├── issued to: Microsoft Root Certificate Authority 2010
+                ├── issued by: Microsoft Root Certificate Authority 2010
+                ├── expiry: Sat Jun 23 22:04:01 2035
+            ├── SHA1 fingerprint: 8BFE3107712B3C886B1C96AAEC89984914DC9B6B
+                ├── issued to: Microsoft Code Signing PCA 2010
+                ├── issued by: Microsoft Root Certificate Authority 2010
+                ├── expiry: Sun Jul 06 20:50:17 2025
```

Comment:
I just found the leaf cert is not the first one. We should make the leaf cert the first one, followed by the intermediate cert and the root ca cert.

Reply:
> I just found the leaf cert is not the first one. We should make the leaf cert the first one, followed by the intermediate cert and the root ca cert.

I also just saw that "The certificate chain starts with your certificate followed by an intermediate one and by root CA certificate" - Will make the changes accordingly